public/app/features/canvas/runtime/scene.tsx

```diff
@@ -838,12 +840,9 @@ export class Scene {
   }
 }
 
-const getStyles = stylesFactory((theme: GrafanaTheme2) => ({
-  wrap: css`
-    overflow: hidden;
-    position: relative;
-  `,
-  selected: css`
-    z-index: 999 !important;
```

Comment:
this is the riskiest change here, did some digging and it appears that this value was never actually being used! The class was being applied to movable's control container that has an overriding z-index of 3000 (vs the 999 being set previously)

[See docs](https://daybrush.com/moveable/release/latest/doc/Moveable.html#className) and pic below
<img width="1388" alt="zindex seems useless" src="https://github.com/grafana/grafana/assets/22381771/82ae369f-cb6d-4a4b-9974-ae81faa6bdcf">